src/components/Regions.js

```diff
@@ -266,7 +273,7 @@ const RegionTable = (props) => {
         )}
         <TableCell align="right" className={classes.operations}>
           <IconButton title="download" onClick={() => handleDownload(item)}>
-            <CloudDownload />
+            <GetApp />
```

Comment:
Sorry to be awkward; can we use `Download` (https://mui.com/material-ui/material-icons/?selected=Download), rather than `GetApp`? As you've seen, `GetApp` can be confusing when you're browsing the code.

It'd be great if you can update the instances in _CustomTable.js_ and _CaptureTable.js_ as well.

Reply:
That makes sense. However, is it OK to install a new package? I think I need to install @mui/icons-material for this icon to work. And of course, I'll update _CutomTable.js_ and _CaptureTable.js_

Reply:
Sorry, that's my fault. I was looking at v5 Material UI, which we're not using yet.
Let's stick with `GetApp` until we've completed migration (under a separate issue).